File: Prepare/Tutorials/30 Days of Code/Javascript (Node.js)/Day 6 - Let's Review.js
function processData(input) {
  //Enter your code here
  let inputCopy = input.split("\n").slice(1);

  inputCopy.forEach((str) => {
    let evenStr = "";
    let oddStr = "";

    for (let i = 0; i < str.length; i++) {
      i % 2 ? (oddStr += str[i]) : (evenStr += str[i]);
    }

    console.log(`${evenStr} ${oddStr}`);
  });
}

process.stdin.resume();
process.stdin.setEncoding("ascii");
_input = "";
process.stdin.on("data", function (input) {
  _input += input;
});

process.stdin.on("end", function () {
  processData(_input);
});

File: Prepare/Tutorials/30 Days of Code/C#/Day 6 - Let's Review.cs
using System;
using System.Collections.Generic;
using System.IO;

class Solution
{
    static void Main(String[] args)
    {
        /* Enter your code here. Read input from STDIN. Print output to STDOUT. Your class should be named Solution */
        int input_itr = Convert.ToInt32(Console.ReadLine());

        for (int j = 0; j < input_itr; j++)
        {
            string input_str = Console.ReadLine();
            string odd_str = "";
            string even_str = "";

            for (int i = 0; i < input_str.Length; i++)
            {
                if (i % 2 == 0)
                {
                    even_str += input_str[i];
                }
                else
                {
                    odd_str += input_str[i];
                }
            }

            Console.WriteLine(even_str + " " + odd_str);
        }
    }
}